CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## Next release
 
+## 0.40.0 / 2020-06-17
+
+* [CHANGE] Update dependencies to prometheus 2.18 (#3231)
+* [CHANGE] Add support for new prometheus versions (v2.18 & v2.19) and fix prom-label-proxy location (#3284)
```

Comment:
(nit) I wouldn't mention the change of location.

Reply:
yeah we tend to only list user-visible changes

Reply:
okay, will remove it then :D